### PR TITLE
add devfolio

### PR DESCRIPTION
**Pull request by Jordanjun98**

### CODE REVIEW

**Comment by benbarth** [comment]
Hi @Jordanjun98, thank you for your contribution. In order for this pull request to be merged, the new entry must be placed in alphabetical order. Also, please update the "how" column to include paraphrased instructions.

File: README.md [hunk 1]
```diff
@@ -48,6 +48,9 @@ In all of the hacktoberfest excitement, many companies want to join the celebrat
 | Uno | ![Shirt](icons/tshirt-greyscale.png) ![Swag](icons/swag-greyscale.png) | "anyone who contributes a pull request to the Uno Platform on GitHub between October 1 and October 31 will receive a Uno Platform T-shirt plus some limited edition swag." | <https://platform.uno/uno-is-joining-hacktoberfest-2019/> (Verified 2019) |
 | Valor | ![Stickers](icons/stickers-greyscale.png) ![Swag](icons/swag-greyscale.png) | 1 PR: pen and sticker. 3 PRs: notepad or wooden phone stand. 5 PRs: pen, sticker, notepad, and a wooden phone stand. | <https://valor-software.com/articles/hacktoberfest-2019-is-coming-and-ngx-bootstrap-strives-for-your-attention.html> (Verified 2019) |
 
+| Devfolio | ![Stickers](icons/stickers-greyscale.png) ![Swag](icons/stickers-greyscale.png) | PR: T-shirt, stickers. | <https://devfolio.co/blog/hacktoberfest-2019-devfolio/> (Verified 2019) |
```
Comment (crweiner) [inline README.md]: You use the stickers image file twice here, so please change one to the t-shirt image. :shirt:

Comment (benbarth) [reply README.md]: (and the image description should reflect the actual image)

**Comment by benbarth** [comment]
Devfolio is already on the list.

**Pull request closed without merging**